pkg/glusterfs/api/types.go

```diff
@@ -102,21 +102,31 @@ type NodeInfo struct {
 	Id string `json:"id"`
 }
 
+type NodeName struct {
+	Hostnames HostAddresses `json:"hostnames"`
+	Zone      int           `json:"zone"`
+}
+
+type Node struct {
+	NodeName NodeName `json:"node"`
+	Devices  []string `json:"devices"`
+}
+
 type NodeInfoResponse struct {
 	NodeInfo
 	State       EntryState           `json:"state"`
-	DevicesInfo []DeviceInfoResponse `json:"devices"`
+	DevicesInfo []DeviceInfoResponse `json:"deviceInfo"`
```

Comment:
Api changes can only happen when absolutely necessary. Here the patch is changing the API.